app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
```

Comment:
(songphi) lỳ gớm
TÌm hiểu I18n và sử dụng cho toàn bộ các file html nhé (hoho) tự self check nhé